restart_process/Tiltfile

```diff
@@ -40,7 +40,10 @@ def _helper(base_ref, ref, entrypoint, live_update, restart_file=RESTART_FILE, *
     # `tilt-restart-wrapper` makes use of `entr` (https://github.com/eradman/entr/) to
     # re-execute $entrypoint whenever $restart_file changes
     if type(entrypoint) == type(""):
-        entrypoint_with_entr = ["/tilt-restart-wrapper", "--watch_file={}".format(restart_file), "sh", "-c", entrypoint]
+        if " " in entrypoint:
+            entrypoint_with_entr = ["/tilt-restart-wrapper", "--watch_file={}".format(restart_file), "sh", "-c", entrypoint]
+        else:
+            entrypoint_with_entr = ["/tilt-restart-wrapper", "--watch_file={}".format(restart_file), entrypoint]
```

Comment:
why this change? I'm hesitant to make it even harder for a user to predict how their command will be run.

Reply:
per the pr description: so that it works even if you don't have `sh` in the container

Reply:
doesn't `live_update + [run('date > {}'.format(restart_file))]` require `sh` though?

Reply:
oof, ya, something was messed up in my test env...maybe we should use a different approach, and just have an assertion that 'sh' and 'date' and 'touch' are all in the container...

Reply:
:( yeahhh. or write our own binary to write some stuff to the specified file, that we can pull into the container alongside restart_process_wrapper? :-/